data/supported-extensions

```diff
@@ -85,6 +85,7 @@ redis 5.5 5.6 7.0 7.1 7.2 7.3 7.4 8.0 8.1 8.2
 seaslog 5.5 5.6 7.0 7.1 7.2 7.3 7.4 8.0 8.1 8.2
 shmop 5.5 5.6 7.0 7.1 7.2 7.3 7.4 8.0 8.1 8.2
 smbclient 5.5 5.6 7.0 7.1 7.2 7.3 7.4 8.0 8.1 8.2
+snappy 5.6 7.0 7.1 7.2 7.3 7.4 8.0 8.1 8.2
```

Comment:
It seems that we can even use snappy on PHP 5.5

Reply:
Making doubt not putting it in lz4 https://github.com/mlocati/docker-php-extension-installer/pull/596/files as well, i thought i checked this locally, but clearly not very well

Reply:
Don't worry, I'll fix it :wink: